### ISSUE TITLE
[Bug] Achievements dont save

### ISSUE THREAD

**Issue by NastyFlytrap** [WerWolv/ImHex#1704]
### Operating System

Windows

### What's the issue you encountered?

I open a file with imhex, i get the achievement the inner workings, i close imhex, i open up a file, and i  get the achievement again. I presume saving the file saves the achivements, but, frankly, the achivements should autosave on their own. 

### How can the issue be reproduced?

Get any achievement, close application, relaunch application, get achievement again

### ImHex Version

1.33.2

### ImHex Build Type

- [ ] Nightly or built from sources

### Installation type

Portable

**Comment by C3pa** [comment]
Is this issue still reproducible? I couldn't reproduce on Windows 10 running ImHex v1.35.4.

**Comment by NastyFlytrap** [comment]
> Is this issue still reproducible? I couldn't reproduce on Windows 10 running ImHex v1.35.4.

1.33.2 is still doing it. I'll try to update and see

Still happening with 1.35.4

**Comment by C3pa** [comment]
Yeah, you are right. Mine are now gone too. IIRC, I was on 1.34.0 and had some achievements there, then I upgraded to 1.35.4 the day I commented on the issue above. They were cleared when I started ImHex earlier today. I unlocked 2 again during that use session and then exited ImHex. Now, opening ImHex again, I can see those 2 unlocked. Looks like the achievements might get cleared only on certain occasions.

**Comment by EamonBrennanDRS** [comment]
I've reproduced this error consistently. Opening ImHex directly from imhex.exe or imhex-gui.exe keeps achievements. Opening a file using ImHex, i.e. Right Click > Open With > ImHex (either .exe or gui.exe) will reset achievements. I set various file extensions to open with both imhex.exe and imhex-gui.exe, and they always reset achievements when I opened them. 

Using Windows 11.

**Comment by art0007i** [comment]
Also happening to me on `imhex-1.35.4-Windows-Portable-x86_64`

I've placed the app in a folder, created a shortcut to `imhex.exe` on my desktop, and drag a random file from my desktop onto the shortcut icon, this causes achievements to get erased every time.

**Comment by GameLord2011** [comment]
Yea, this is starting to get annoying

**Comment by paxcut** [comment]
@GameLord2011 what's the issue you are running into? Achievements should be saving now and the popups can be turned off. Make sure you are running on a nightly build.

**Comment by GameLord2011** [comment]
It is not saving it in the projects.

**Comment by paxcut** [comment]
Achievements are not saved in projects but globally. If you are using version 1.37.4 (check Help>About) then achievements will not save at all. you need to upgrade to a nightly build (1.38.0) for achievements to be properly saved. Alternatively you can turn off the achievement popups in the Extras>Achievements window.

**Comment by GameLord2011** [comment]
Got it
